doc/classes/EditorNode3DGizmo.xml

```diff
@@ -43,7 +43,7 @@
 			<param index="0" name="id" type="int" />
 			<param index="1" name="secondary" type="bool" />
 			<description>
-				Override this method to return the name of an edited handle (handles must have been previously added by [method add_handles]). Handles can be named for reference to the user when editing.
+				Override this method to return the name of an edited handle (handles must have been previously added by [method add_handles]). Handles can be named for reference to the user when editing. By convention, handle names should use [i]Sentence case[/i] (e.g. [code]"Inner radius"[/code] instead of [code]"Inner Radius"[/code]).
```

Comment:
Genuinely the first time "Sentence case" would be ever referenced in the documentation. I would consider spelling out what it means, but if it sounds wrong, I guess this is okay.
This remark also fits quite snuggly as a note.
```suggestion
				Override this method to return the name of an edited handle (handles must have been previously added by [method add_handles]). Handles can be named for reference to the user when editing.
				[b]Note:[/b] By convention, the first word of the name should be capitalized, while the rest should be in lowercase. For example, [code]"Inner radius"[/code].
```
Or similar?